queries.sql

```diff
@@ -53,4 +53,29 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) 
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+-- Alter to add new column
+ALTER TABLE animals
+ADD COLUMN species varchar(255);
+
+-- transaction 1
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = 'unspecified';
+COMIT;
+
```

Comment:
- [ ] these are supposed to be 2 separate transactions.
- after setting the species column to unspecified you're supposed to verify if the changes are made
- then you should rollback the changes
- and verify if the changes were undone.

- [ ] line 65 and 66 should be in a separate transaction
- after updating the columns you're supposed to verify if the changes are made
- after that commit the changes 
- and after commit check if the changes persist.